registry-repo-readmes/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"sort"
+	"time"
+
+	"chainguard.dev/sdk/auth/token"
+	registry "chainguard.dev/sdk/proto/platform/registry/v1"
+	"chainguard.dev/sdk/sts"
+	"github.com/gomarkdown/markdown"
+	"github.com/microcosm-cc/bluemonday"
+)
+
+const (
+	group    = "chainguard"
+	audience = "https://console-api.enforce.dev"
+	issuer   = "https://issuer.enforce.dev"
+	identity = "720909c9f5279097d847ad02a2f24ba8f59de36a/a033a6fabe0bfa0d"
```

Comment:
is it okay to add this?

Reply:
i got this from here: https://github.com/chainguard-dev/platform-examples/blob/main/image-diff/main.go#L61